resources/snap_packs/attributes/snap_pack_id.yml

```diff
@@ -0,0 +1,5 @@
+type: string
+
+description: Unique identifier prefixed with `ord_`.
+
+pattern: "^ord_[0-9a-f]{26}$"
```

Comment:
Snap packs are only available via CMB. Orders created in CMB are prefixed with `ord_`